### DISCUSSION TITLE
Is there any way (batch, mult. replace) to only replace words that aren't at the beginning of a sentence?

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5118] [General]
I'm editing The Sopranos. They always capitalize the word "Internet" and I hate it. I want to batch all the occurrences to lowercase, but don't want to accidentally change the case of any times they use the word Internet at the beginning of a sentence. Is there a regular expression for this?

**Comment by OmrSi** [comment]
You can use `(?<!^)Internet`

**Reply by OmrSi** [reply]
Although, this wouldn't change lines like:
```
I love the
Internet.
```

**Reply by OmrSi** [reply]
This could be a better option: `(?<=\w+[^?!.]\s*)Internet`
It looks for the word `Internet` but only if there is another word before it.

**Comment by DrReddP** [comment]
Excellent. Thank you
